testing/unittests/unittests.cpp

```diff
@@ -569,5 +570,71 @@ class StringBufferTest : public CppUnit::TestFixture
 CPPUNIT_TEST_SUITE_REGISTRATION( StringBufferTest );
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( StringBufferTest, "StringBufferTest" );
 
+StringBuffer &mbToBase58(StringBuffer &s, const MemoryBuffer &data)
+{
+    size_t b58Length = data.length() * 2 + 1;
+
+    if (b58enc(s.reserve(b58Length), &b58Length, data.toByteArray(), data.length()) && b58Length > 1)
```

Comment:
Should this fail if b58enc returns false, or clear s(), otherwise you might have examples which accidentally work.